roles/ssp/tasks/configure-common.yml

```diff
@@ -26,6 +26,41 @@
   tags:
     - ssp:config:authsources
 
+
+
+
+- name: Copy SP SSL certificate (.crt)
+  copy:
+    content: '{{ item.sp_ssl_certificate }}'
+    dest: "{{ ssp_certdir }}/sp-sso.crt"
+    owner: "{{ ssp_webuser }}"
+    group: "{{ ssp_webgroup }}"
+    mode: "0644"
+  with_items: "{{ ssp_authsources_saml }}"
+  when: item.generate_ssl_certificate|bool == False
+  become: yes
+  no_log: yes
+  tags:
+    - ssp:config:authsources
+
+
+- name: Copy SP SSL certificate (.key)
+  copy:
+    content: '{{ item.sp_ssl_certificate }}'
+    dest: "{{ ssp_certdir }}/sp-sso.key"
```

Comment:
Change `sp-sso.key` to `sp-{{ item.name }}.key`. See also https://github.com/rciam/rciam-deploy/blob/5e4c2b009e4c6d7a5db3becd0f8584a2c9497a8b/roles/ssp/tasks/configure-common.yml#L19